edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Diagnostics.Test/Microsoft.Azure.Devices.Edge.Agent.Diagnostics.Test.csproj

```diff
@@ -11,10 +11,6 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.4.0" />
     <PackageReference Include="Moq" Version="4.10.1" />
-    <PackageReference Include="StyleCop.Analyzers" Version="1.1.118">
```

Comment:
where is this duplicated?

Reply:
https://github.com/Azure/iotedge/blob/438c7c8a33cdbf9a58a182501189a4d5aa9142b0/stylecop.props#L3